tests/test_charities.py

```diff
@@ -0,0 +1,2 @@
+import unittest
+import json
```

Comment:
I know, I know. 

Need to do further research into mocks/stubs in Python/Flask to write API tests.

Reply:
If we have time we'll come back to the testing. It's a little bit harder than the other tests we wrote. Even if it's later down the road we can figure it out. :)